### DISCUSSION TITLE
Difference between `eventbox` and `button`

### DISCUSSION THREAD

**Discussion by jaeheonji** [elkowar/eww#566] [Q&A] [answered]
Hi! I have one thing that confuses me.
`eventbox` and `button` seem to have similar widgets. Could you tell me the difference between these two?
For most simple widgets, I can create the same Item using both widgets. In this case, which one is the standard to use?

**Comment by viandoxdev** [comment, marked as answer]
Eventbox is more general than button and support additional events (as well as css pseudo selectors). It mostly comes down to style (a button is more explicit) as well as keyboard support (tab and enter) though I'm not too sure about how it's handled in eww.

**Reply by jaeheonji** [reply]
Thank you!
